src/index.js

```diff
@@ -96,7 +96,11 @@ function wpDie( message = '' ) {
 
 function wpDieTemplate( template, error ) {
```

Comment:
Can you please add a default value for the error parameter if we are going to make it optional? `HtmlService` templates quietly die if you reference an unset property.